src/index.ts

```diff
@@ -278,9 +278,7 @@ export class Supertab {
       paymentModel: "pay_later",
     });
 
-    const filterStatuses: TabStatus[] = [TabStatus.Open, TabStatus.Full];
-
-    if (filterStatuses.includes(tab?.status)) {
+    if (tab?.status === TabStatus.Open) {
```

Comment:
@jmcodingde I know absolutely nothing about TS, but I feel this change will only include `OPEN` tab, is it correct? What about "legacy" tabs (those which have been closed but never paid) ?

Reply:
Reverted that change here: [4bb919f](https://github.com/laterpay/supertab-browser/pull/75/commits/4bb919f4eb11339d73f0a1c5e9e49e5ba6d56a3a)